tests/config.json

```diff
@@ -16,6 +16,10 @@
       "model_name": "microsoft/phi-2",
       "model_class": "PhiForCausalLM"
     },
+    {
+      "model_name": "microsoft/Phi-3-mini-4k-instruct",
+      "model_class": "Phi3ForCausalLM"
```

Comment:
Add extra parameter named `num_layers`, in case it's provided it will be used in tests, if not provided default value will be 1.

Reply:
to be done in PR #79